### PR TITLE
Fix: changes in search, footer, summary  and browse pages

### PR DESCRIPTION
**Pull request by SirineMhedhbi**
## Context

See : https://docs.google.com/spreadsheets/d/1lEdrxJKKOJYbSYH_Ptzu2dUliHAKcFTirrw8vs9Hcuk/edit#gid=713725591

## Changes
- [x] Update contacts text to be like this:   For additional information, contact Enrico Motta (enrico.motta@open.ac) and Aliaksandr Birukou (Birukou@rien.com).
image

- [x] Remplace in submsssion table, “Uploaded” by “Submitted”
image

- [x]  remove create “Initially created On” custom label for release date 
image

- [x] Update  links of support in topnav to be as footer
image

- [x] Fixing missing translation in the footer
image

- [x] Change “uploaded” to “submitted “, and “release” to “creation”
image

- [x] Fix the buf in search page to open advanced options by default
image

### CODE REVIEW

File: app/helpers/ontologies_helper.rb [hunk 1]
```diff
@@ -123,8 +123,14 @@ def agent?(sub_metadata, attr)
   end
 
   def display_contact(contacts)
-    contacts.map { |c| "#{c.name.humanize} at #{c.email}" if c.member?(:name) && c.member?(:email) }&.join(", ")
-  end
+    contacts.map do |c|
+      next unless c.member?(:name) && c.member?(:email)
+  
+      formatted_name = c[:name].split.map(&:capitalize).join(' ')
```
Comment (syphax-bouazzouni) [inline app/helpers/ontologies_helper.rb:129]: try .titlize  https://apidock.com/rails/String/titleize

File: app/views/layouts/_topnav.html.haml [hunk 2]
```diff
@@ -54,13 +54,10 @@
             - s.item do
               = link_to(t('layout.header.submit_feedback'), feedback_path(location: encode_param(request.url)), id: "submitFeedbackMenuItem", class: "pop_window")
           - d.with_section do |s|
-            - s.header do
-              = t('layout.header.documentation')
             - s.item do
-              = link_to(t('layout.header.help'), "#{$WIKI_HELP_PAGE}", target: "_blank")
+              = link_to(t('layout.header.documentation'), "https://ontoportal.github.io/documentation/", target: "_blank")
```
Comment (syphax-bouazzouni) [inline app/views/layouts/_topnav.html.haml:58]: this need to use the configuration, not been hard coded

File: app/views/layouts/_topnav.html.haml [hunk 3]
```diff
@@ -54,13 +54,10 @@
...
             - s.item do
-              = link_to(t('layout.header.release_notes'), "#{$RELEASE_NOTES}", target: "_blank")
+              = link_to(t('layout.header.cite_us'), "https://doc.jonquetlab.lirmm.fr/share/e6158eda-c109-4385-852c-51a42de9a412/doc/publications-and-references-87tEoeoGKy", target: "_blank")
```
Comment (syphax-bouazzouni) [inline app/views/layouts/_topnav.html.haml:60]: this need to use the configuration, not been hard coded

File: app/views/layouts/_topnav.html.haml [hunk 4]
```diff
@@ -54,13 +54,10 @@
...
             - s.item do
-              = link_to(t('layout.header.publications'), $PUBLICATIONS_URL, target: "_blank")
-
+              = link_to(t('layout.header.release_notes'), "https://doc.jonquetlab.lirmm.fr/share/e6158eda-c109-4385-852c-51a42de9a412/doc/release-notes-btKjZk5tU2", target: "_blank")
```
Comment (syphax-bouazzouni) [inline app/views/layouts/_topnav.html.haml:62]: this need to use the configuration, not been hard coded

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]

File: app/helpers/ontologies_helper.rb (hunk 1 above)
Comment (SirineMhedhbi) [reply app/helpers/ontologies_helper.rb]: done

File: app/views/layouts/_topnav.html.haml (hunk 2 above)
Comment (SirineMhedhbi) [reply app/views/layouts/_topnav.html.haml]: done

File: app/views/layouts/_topnav.html.haml (hunk 3 above)
Comment (SirineMhedhbi) [reply app/views/layouts/_topnav.html.haml]: done

File: app/views/layouts/_topnav.html.haml (hunk 4 above)
Comment (SirineMhedhbi) [reply app/views/layouts/_topnav.html.haml]: done

**Review by syphax-bouazzouni — APPROVED** [review]

**Comment by syphax-bouazzouni** [comment]
@SirineMhedhbi you will need to update the configuration repository to update `.en` file as here.

**Pull request merged**